wire-ios-sync-engine/Source/UserSession/ZMUserSession+Authentication.swift

```diff
@@ -45,8 +45,9 @@ extension ZMUserSession {
 
     public var isLoggedIn: Bool { // TODO jacob we don't want this to be public
         let needsToRegisterClient = ZMClientRegistrationStatus.needsToRegisterClient(in: managedObjectContext)
+        let needsToRegisterMLSClient = ZMClientRegistrationStatus.needsToRegisterMLSClient(in: managedObjectContext)
 
-        return isAuthenticated && !needsToRegisterClient
+        return isAuthenticated && !needsToRegisterClient && !needsToRegisterMLSClient
```

Comment:
Does this mean that clients with only Proteus would not be logged in?

Reply:
Only if also need register an MLS client (controlled via feature config).